src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/Services/SyntaxFacts/CSharpSyntaxFacts.cs

```diff
@@ -732,11 +729,7 @@ EnumMemberDeclarationSyntax or
     }
 
     public bool IsTopLevelNodeWithMembers([NotNullWhen(true)] SyntaxNode? node)
-    {
-        return node is BaseNamespaceDeclarationSyntax or
-               TypeDeclarationSyntax or
-               EnumDeclarationSyntax;
-    }
+        => node is BaseNamespaceDeclarationSyntax or BaseTypeDeclarationSyntax;
```

Comment:
>BaseNamespaceDeclarationSyntax or BaseTypeDeclarationSyntax; [](http://example.com/codeflow?start=19&length=60)

Is this change intended for this PR?

Reply:
Yes.  Was touchign the 'AddMembers' codepaths to update them to use this, and it hits thsi method.  so i fixed it at the same time.

Reply:
I'm a little confused by this change. Is EnumDeclarationSyntax not a top level node, or not one with members?

Reply:
EnumDecl and TypeDecl are the only subclasses of BaseTypeDeclarationSyntax.  So we already have a single type that checks for both.  we don't need two type checks.